x-pack/plugins/siem/cypress/objects/case.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
for the future, maybe you could reuse some of the existing case objects. not necessary, just trying to save time if we can:
`x-pack/plugins/case/server/routes/api/__fixtures__/mock_saved_objects.ts`
`x-pack/legacy/plugins/siem/public/containers/case/mock.ts`


Maybe if you find these useful, we can combine and put into a file in plugins/case/common so we can use everywhere

Reply:
I like the idea of reusing the test objects that we already have.

Reply:
cool i put it as a part of this follow up ticket https://github.com/elastic/siem-team/issues/631